app/views/editor/rich.html.erb

```diff
@@ -29,8 +29,6 @@
 
     <!-- errors module -->
     <div class="ple-module container">
```

Comment:
```diff
- <div class="ple-module container">
```
Remove this line too

Reply:
All right. Removing the closing `div ` as well